index.js

```diff
@@ -26,7 +26,15 @@ function lint(textEditor) {
 	const defaultCwd = process.cwd();
 	process.chdir(dir);
 
-	const pkg = loadJson(path.join(dir, 'package.json'));
+	let pkg = loadJson(path.join(dir, 'package.json'));
+
+	// get the parent `package.json` if there's a `"xo": false` in the current one
+	if (pkg.xo !== undefined) {
+		while (pkg.xo === false && dir !== null) {
+			dir = pkgDir.sync(path.join(dir, '..'));
+			pkg = dir === null ? pkg : loadJson(path.join(dir, 'package.json'));
+		}
+	}
 
```

Comment:
If a package.json has `"xo": false`, but no other package.json up the directory tree, it will use the one with `"xo": false`, which is clearly wrong.